### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by atinux** [nuxt/nuxt#9287] [Kind words]
I am so grateful to have such a vibrant community, contributors and framework team 💚

**Comment by kissu** [comment]
Thanks to all the people behind the framework that helps us building such nice apps. 🎊 🙏🏼

**Comment by flozero** [comment]
Thank to all your works !

**Comment by duncandejong** [comment]
Nuxt 2 was good... But i am amazed by all the features and effort put into Nuxt 3.
